config/routes.rb

```diff
@@ -22,5 +22,6 @@
   end
 
   get '/campaigns', to: 'static#campaigns'
+  get '/mapathon', to: 'static#mapathon'
```

Comment:
same as above, use plural form as it's getting a "collection" of mapathons, and use "events" as it's more generic.